Cargo.toml

```diff
@@ -24,6 +24,7 @@ num = "*"
 rustc-serialize = "*"
 lazy_static = "*"
 log = "*"
+clap="*"
```

Comment:
okay, I'll change it, but why do you prefer `docopt` does it have any extra features over `clap` ?